flex/http.py

```diff
@@ -83,10 +91,16 @@ def data(self):
         elif self.body is EMPTY:
             return EMPTY
         elif self.content_type.startswith('application/json'):
-            if type(self.body) == bytes:
-                return json.loads(self.body.decode('utf-8'))
-            else:
-                return json.loads(self.body)
+            try:
+                if type(self.body) == bytes:
```

Comment:
Sure, this was like this before however - is this just a while-your-here-may-as-well situation?